pkg/k8s/controllers/telemetryservice_controller.go

```diff
@@ -31,9 +38,16 @@ type TelemetryServiceReconciler struct {
 	Scheme *runtime.Scheme
 }
 
+var tsNamespacedName = types.NamespacedName{
```

Comment:
can we use `req.NamespacedName` to replace it?

Reply:
Actually I refer to `telemetryservice_install.yaml` to use the namespace and name, if we create service and deployment automatically, Should we use the same namespace and name or specify ones?